src/plugins/data_view_management/kibana.json

```diff
@@ -5,6 +5,7 @@
   "ui": true,
   "requiredPlugins": ["management", "data", "urlForwarding", "dataViewFieldEditor", "dataViewEditor", "dataViews", "fieldFormats"],
   "requiredBundles": ["kibanaReact", "kibanaUtils"],
+  "optionalPlugins": ["spaces"],
```

Comment:
That's another great example of why it was worth splitting data views into multiple smaller plugins 👍  Really helps with dealing with circular dependencies. I am sure we would get one if we'd try to add `space` to base `dataViews` plugin